Libs/Python/ShapeworksPython.cpp

```diff
@@ -1328,6 +1329,100 @@ PYBIND11_MODULE(shapeworks_py, m)
        "return the variance accounted for by the principal components")
   ;
 
+  py::class_<ReconstructSurface<ThinPlateSplineTransform>>(m, "ReconstructSurface_ThinPlateSplineTransform")
```

Comment:
These class names are really not long enough. How thin are the plates of the spline? What color is the surface? Are they using tractors or shovels?! </sarcasm>